extensions/mongodb/src/main/java/com/hazelcast/jet/mongodb/MongoSourceBuilder.java

```diff
@@ -254,15 +256,26 @@ private Batch(
         }
 
         /**
-         * If {@code true}, the lack of database or collection will cause an error.
-         * If {@code false}, database and collection will be automatically created.
-         * Default value is {@code true}.
+         * If non {@link ResourceChecks#NEVER}, the lack of database or collection will cause an error.
+         * Otherwise, database and collection will be automatically created.
```

Comment:
too many negations here (if non + otherwise, lack of). Could you reword it to make it simpler?
"@param checkResourceExistence if exception should be thrown" -  this is not boolean ("if") but enum/mode

Reply:
7831956e3b2ce0787afc566a54b89fa469b724b0